src/languageservice/parser/yaml-documents.ts

```diff
@@ -137,7 +137,7 @@ export class SingleYAMLDocument extends JSONDocument {
       if (!range) {
         return;
       }
-      const diff = range[2] - offset;
+      const diff = range[1] - offset;
```

Comment:
what could happened if next line hasn't comment?

Reply:
Tested this manually and it works. Added another unit test for this situation.